appknox/reports.go

```diff
@@ -137,3 +137,17 @@ func (s *ReportsService) WriteReportDataToFile(respBody io.ReadCloser, outputFil
 	_, err = io.Copy(out, respBody)
 	return outputFilePath, err
 }
+
+func (s *ReportsService) CreateReport(ctx context.Context, fileID int) (reportID int, err error) {
+	url := fmt.Sprintf("api/v2/files/%d/reports", fileID)
+	request, err := s.client.NewRequest("POST", url, nil)
+	var reportResult ReportResult
+	resp, err := s.client.Do(ctx, request, &reportResult)
+	if resp != nil && resp.StatusCode == 400 {
+		return 0, errors.New(resp.Status)
+	}
+	if err != nil {
+		return 0, err
+	}
+	return reportResult.ID, nil
```

Comment:
I think CreateReport should return the whole ReportResult instance. It should be up to helper or cmd to decide what to take out of that instance.

Reply:
Done 👍🏼